package.json

```diff
@@ -33,6 +33,7 @@
     "check-types": "lerna run check-types",
     "toolpad": "node --enable-source-maps packages/toolpad-app/cli.mjs",
     "jsonSchemas": "tsx ./scripts/docs/generateJsonSchemas.ts",
+    "updateMonorepo": "tsx ./scripts/updateMonorepo.ts",
     "check-changes": "git add -A && git diff --exit-code --staged",
```

Comment:
We don't seem to have settled on one naming convention for commands.

Reply:
oops, now we do 🙂 thanks